File: SalesTaxes/Controller/ReceiptViewController.m
//
//  ReceiptViewController.m
//  SalesTaxes
//
//  Created by Fabrizio Duroni on 13/11/2016.
//  Copyright © 2016 Fabrizio Duroni. All rights reserved.
//

#import "ReceiptViewController.h"

@interface ReceiptViewController ()

/// Receipt content label.
@property (weak, nonatomic) IBOutlet UILabel *receiptContentLabel;

@end

@implementation ReceiptViewController

- (void)viewDidLoad {
    
    [super viewDidLoad];

    //Create receipt.
    self.receiptContentLabel.text = [self.receipt generateReceipt];
}

@end

File: SalesTaxesSwift/Controller/ReceiptViewController.swift
//
//  ReceiptViewController.swift
//  SalesTaxes
//
//  Created by Fabrizio Duroni on 21/11/2016.
//  Copyright © 2016 Fabrizio Duroni. All rights reserved.
//

import UIKit

class ReceiptViewController: UIViewController {
    
    /// Receipt content label.
    @IBOutlet weak var receiptContentLabel: UILabel!
    /// Receipt object model.
    var receipt:Receipt?
    
    override func viewDidLoad() {
        
        super.viewDidLoad()
        
        self.receiptContentLabel.text = self.receipt?.generateReceipt()
    }

    override func didReceiveMemoryWarning() {
        
        super.didReceiveMemoryWarning()
    }
}
